doc/DeveloperGuide.md

```diff
@@ -94,7 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
-
+5. Should be able to be used by any individual who has not used an address book before.
+6. Should not showcase any unnecessary and explicit content which is unrelated to the address book.
```

Comment:
What does this mean?
Please try be very specific when writing NFRs

Reply:
Yes I will try to be more specific in the upcoming NFRs in my addressbook level4. Thank you for the recommendation!